internal/stack/certs.go

```diff
@@ -140,9 +145,18 @@ func initServiceTLSCertificates(ca *certs.Issuer, caCertFile string, certFile, k
 		return certs.LoadCertificate(certFile, keyFile)
 	}
 
-	cert, err := ca.Issue(certs.WithName(service))
-	if err != nil {
-		return nil, fmt.Errorf("error initializing certificate for %q", service)
+	var cert *certs.Certificate
+	var err error
+	if service == "elastic-agent" {
```

Comment:
This special case is too hidden here, maybe `tlsServices` could be a struct, so we can define it like this:
```
var tlsServicesServerless = []tlsService{
	{Name: "logstash"},
	{Name: "elastic-agent", IsClient: true},
}
```
And then here:
```suggestion
	if service.IsClient {
```
WDYT?

Reply:
Looks clean and readable this way.. Will test first by removing `ssl_client_authentication` from agent input in logstash config and if it is still needed then I ll modify this

Reply:
Will fix this